modules/acquia_cms_headless/src/Plugin/AcquiaCmsHeadless/HeadlessApiKeys.php

```diff
@@ -227,11 +226,14 @@ public function createOperationLinks(string $entityType): array {
           ];
         }
         else {
-          $route_name = $entity->toUrl($operationLink['route'])->getRouteName();
-          $operation[$key] = [
-            'url' => Url::fromRoute($route_name, ['consumer' => $entity->id()], $destination),
-            'title' => $operationLink['title'],
-          ];
+          // Check link templete.
```

Comment:
@chandan-singh7929 Can we create separate method for this code block in the acquia_cms_headless module?

cc: @vishalkhode1

Reply:
Created separate ticket for refactor acquia_cms_headless module : https://acquia.atlassian.net/browse/ACMS-3467